src/exponential.jl

```diff
@@ -97,7 +97,40 @@ function expm_overapproximation(A::IntervalMatrix{T, Interval{T}}, t, p) where {
     return res
 end
 
+# Implementation of Prop. 1 in Althoff, Matthias, Bruce H. Krogh, and Olaf Stursberg.
+# "Analyzing reachability of linear dynamic systems with parametric uncertainties."
+# Modeling, Design, and Simulation of Systems with Uncertainties. Springer, Berlin, Heidelberg, 2011. 69-94.
 function _expm_remainder(A::IntervalMatrix{T}, t, p; n=checksquare(A)) where {T}
+    C = max.(abs.(inf.(A)), abs.(sup.(A)))
+    M = exp(C*t)
+
+    # compute Q = I + Mt + M^2/2! + ... + M^p/p!
+    # where M = exp(abs(A*t)) and abs is taken component-wise
+    Q = IntervalMatrix(zeros(Interval{T}, n, n))
+    @inbounds for i in 1:n
+        Q[i, i] += one(T)
+    end
+
+    tⁱ = 1
+    i! = 1
+    Mⁱ = copy(M)
+    for i in 1:p
+        i! *= i
+        tⁱ *= t
+        Q = Q + Mⁱ * tⁱ/i!
+        Mⁱ = Mⁱ * M
+    end
+    Y  = M - Q
+    Γ = IntervalMatrix(fill(zero(T)±one(T), (n, n)))
+    E = Γ * Y
+    return IntervalMatrix(E) # TODO why do we need the IntervalMatrix?
```

Comment:
> why do we need the IntervalMatrix?

In the past this happened to me and I found that at some point an `IntervalMatrix` used an operation that is not properly wrapped as an `IntervalMatrix` again. I guess if you step through the algorithm line by line and print the result, you should find the place where you change to an `Array{Interval, 2}`. (However, as a word of warning, sometimes an `IntervalMatrix` is also printed as an `Array{Interval, 2}`. So always be sure to print `typeof(·)`.)

Reply:
Forget what I said. In this case you do `Γ::IntervalMatrix * Y::Matrix`, which we do not define, so I guess the Julia default of `*` interprets `Γ` as a matrix. The proper solution is to define `*(::IntervalMatrix, ::AbstractMatrix)`.